src/style.css

```
@@ -0,0 +1,127 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card-product {
+  position: relative;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 14px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-product:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.card-product__imac_img {
+  display: flex;
+  justify-content: center;
+  margin: 15px 5px 45px;
+  height: 134px;
+  width: 160px;
+}
+
+.card-product__name {
+  margin: 0 0 6px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+}
+
+.card-product__code {
+  margin: 0 0 15px;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 13px;
+  color: #000;
+}
+
+.card-product__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 20px;
+}
+
+.card-product__stars {
+  display: flex;
+  align-items: center;
+
+}
+
+.stars__star {
+  display: flex;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card-product__stars .stars__star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
```

Add modifier and corresponding styles to Stars block

Your site should support all possible star modifiers stars--1, stars--2 etc.

```suggestion
.stars--1 > .stars__star:nth-child(-n+1),
.stars--2 > .stars__star:nth-child(-n+2),
.stars--3 > .stars__star:nth-child(-n+3),
.stars--4 > .stars__star:nth-child(-n+4),
.stars--5 > .stars__star:nth-child(-n+5) {
  background-image: url("images/star-active.svg");
}
```